### PR TITLE
Add modals

### PR DESCRIPTION
**Pull request by vzajkov**
## Description
Add modals to drop down on form submit. A success modal drops down on successful submit, and an error modal should drop down on a failed submit. Modals can be closed manually and close with a CSS transition. 

Once down, modals can be closed by pressing the X on the top right of the modal, or by clicking anywhere in the form. 

## Test Plan
1. Click 'Verify Next Resource`
2. Submit Form
3. Scroll to top to see modal.

### CODE REVIEW

**Comment by karenmrose** [comment]
@vzajkov Can you merge in the latest master? Also, we no longer need to commit the latest bundle file (it will be created locally and also is created in Heroku, but should not be checked into GitHub anymore). Can you make sure that file gets deleted?

**Comment by vzajkov** [comment]
@karenmrose Merged in the latest master, no longer commiting the bundle file. Wondering about the Spacer in the ResourceView - is it necessary? Asking because it currently interferes with the modals disappearing all the way.

**Comment by karenmrose** [comment]
@vzajkov It looks like the bundle file is still showing up in the diff...

**Comment by vzajkov** [comment]
Getting a `error  HTML entities must be escaped  react/no-unescaped-entities` on Travis, although no extra tags in the file.

**Comment by karenmrose** [comment]
@vzajkov If you run `yarn lint` on this branch, you'll see which lint errors need to be fixed.

**Comment by karenmrose** [comment]
@vzajkov Can you remove the logic that shows the alert? We won't need that anymore.

screen shot 2018-10-16 at 7 29 29 pm

File: client/js/verification-portal/pregnancy-resource-center/form.js [hunk 1]
```diff
@@ -325,6 +325,8 @@ const VerificationPortalForm = reduxForm({
 	form: 'verificationPortal',
 })(VerificationPortalFormClass)
 
+// const selector = formValueSelector('verificationPortal')
```
Comment (karenmrose) [inline client/js/verification-portal/pregnancy-resource-center/form.js]: @vzajkov Do we need this or can this file be reverted?

File: client/js/verification-portal/components/header-modal.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+		'font-size': '30px',
+		'font-weight': 'bold',
+		'letter-spacing': '1.18px',
+		'line-height': '27px',
+		'font-family': 'Century Gothic, sans-serif !important',
+		'padding-left': '10px',
+		overflow: 'hidden',
+		transition: ['height'],
+		transitionDuration: '2s',
+		// 'animation-name': 'closing-animation',
```
Comment (karenmrose) [inline client/js/verification-portal/components/header-modal.js]: @vzajkov Do you mind removing any commented out code that we don't need?

File: client/js/verification-portal/components/header-modal.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+const styles = {
+	root: {
+		height: '0px',
+		color: '#FFFFFF',
+		display: 'flex',
+		'text-align': 'left',
+		background: '#ee6298',
+		'justify-content': 'center',
+		'align-items': 'center',
+		'font-size': '30px',
```
Comment (karenmrose) [inline client/js/verification-portal/components/header-modal.js]: It looks like the heading (1st line) text should be `37px` and the 2nd line should be `20px`

**Comment by karenmrose** [comment]
@vzajkov It's looking good! 👏 Just left a few comments.

File: client/js/hah-app/reducers.js [hunk 4]
```diff
@@ -55,11 +58,27 @@ const miniAppReducer = (state = {}, action) => {
 	}
 }
 
+const localStateReducer = (state = {}, action) => {
+	switch (action.type) {
+		case FORM_SUCCESSFULLY_SUBMITTED:
+			return {
+				...state,
+				setFormStatus: action.formStatus,
```
Comment (karenmrose) [inline client/js/hah-app/reducers.js]: What do you think of calling this state `formStatus`?

File: client/js/hah-app/reducers.js [hunk 5]
```diff
@@ -55,11 +58,27 @@ const miniAppReducer = (state = {}, action) => {
...
+			}
+		default:
+			return {
+				...state,
+				setFormStatus: 'Pending',
```
Comment (karenmrose) [inline client/js/hah-app/reducers.js]: What do you think of calling this state `formStatus`?

**Comment by karenmrose** [comment]
Just a few more minor wording changes! See comments in this PR and also in Slack.

**Pull request merged**